app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,59 @@
+class OrdersController < ApplicationController
+  before_action :load_id, only: [:commit_order, :show, :remove_item, :payment]
+  # GET order new
```

Comment:
fix path
to check paths, type `rails routes` in terminal

Reply:
GET